<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMapAppAdvClass</class>
 <widget class="QMainWindow" name="MyMapAppAdvClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMapAppAdv</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,0">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="layout_1" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="layout_2" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="layout_3" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="layout_4" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mymapappadv.qrc"/>
 </resources>
 <connections/>
</ui>
